docs/sources/clients/promtail/gcplog-cloud.md

```diff
@@ -69,3 +69,19 @@ We need a service account with following permissions.
 - pubsub.subscriber
 
 This enables promtail to read log entries from the pubsub subscription created before.
+
+## Operations
+
+Sometimes you may have to check how much logs you are writting to the pubsub topic (mostly for cost estimation).
+
+These messages stays in Pubsub Subscription till its acknowledged. Following command just removes those log messages without having it to be consumed via promtail or any other pubsub consumer.
```

Comment:
Can these alternatively be configured via a TTL?

Reply:
True. we do have message-retention-duration for the specific subscription. But it usually be 7 or 10 days in production. And we can change of course during creation of the subscription.

But this operation steps is to inform there is another way to clear those messages rather than waiting till message-retention-duration.

Reply:
I just looked and you already have that retention config covered in the doc. LGTM